packages/cli/src/commands/fix/tasks/convert-task.ts

```diff
@@ -44,14 +44,18 @@ export function convertTask(
 
       // this just cares about ts files which are already in the proper migration order
       if (orderedFiles) {
+        // Filter out all files that outside of the target directory
+        const filesToMigrate = ctx.orderedFiles.filter((file) =>
+          file.startsWith(resolve(options.rootPath, targetPath))
```

Comment:
I think we need to make `targetPath` absolute in the first place right in cli's `.action` since we do this conversion in a lot of places.